clients/web/test/spec/collectionSpec.js

```diff
@@ -304,6 +304,24 @@ describe('Test collection actions', function () {
             return $('#g-confirm-button:visible').length > 0;
         }, 'delete confirmation to appear');
 
+        waitsFor(function () {
```

Comment:
Can you also add a test for deleting with no value entered, since we have a code path for that error?

Reply:
This test has to try to delete a collection without enter any value, but it cannot succeed right ?

Reply:
Yes.